.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: csharp
-
+mono: latest
+dotnet: 2.1.4
```

Comment:
Does this require `2.1.4`? If not, shouldn't we keep it as low as possible or use a matrix to test multiple versions?

Reply:
Agreed, I'd guess 2.0.3 is the lowest possible.